corehq/apps/hqadmin/service_checks.py

```diff
@@ -96,13 +96,19 @@ def check_elasticsearch():
 
     doc = {'_id': 'elasticsearch-service-check-{}'.format(random_hex()[:7]),
            'date': datetime.datetime.now().isoformat()}
-    send_to_elasticsearch('groups', doc)
-    refresh_elasticsearch_index('groups')
-    hits = GroupES().remove_default_filters().doc_id(doc['_id']).run().hits
-    send_to_elasticsearch('groups', doc, delete=True)  # clean up
-    if doc in hits:
-        return ServiceStatus(True, "Successfully sent a doc to ES and read it back")
-    return ServiceStatus(False, "Something went wrong sending a doc to ES")
+    try:
+        send_to_elasticsearch('groups', doc)
+        refresh_elasticsearch_index('groups')
+        hits = GroupES().remove_default_filters().doc_id(doc['_id']).run().hits
+    except Exception:
+        raise
+    else:
```

Comment:
The above three lines can be removed.

Reply:
Yeah there's a minor difference, but I suppose it doesn't matter much. I'll go ahead and remove